client/src/app/project-create-edit/project-create-edit.module.ts

```diff
@@ -0,0 +1,56 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ProjectMetadataFormComponent } from './components/project-metadata-form/project-metadata-form.component';
+import { EditProjectComponent } from './pages/edit-project/edit-project.component';
+import { CreateProjectComponent } from './pages/create-project/create-project.component';
```

Comment:
Please have a consistent spacing in the import statement.

Reply:
Please see #110. I made a seperte PR as I've optimized imports across all files